dts/bindings/dma/dma.yaml

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2019, Linaro Limited
```

Comment:
This binding is not needed as following properties are now part of base.yaml and hence available to all bindings.